v1/linkml-schemas/evidence.yaml

```diff
@@ -141,10 +141,12 @@ enums:
       multiple_choice:
   FraudCheckType:
     permissible_values:
-      mortality_check:
+      country_data:
+        description: Denotes if a user has an international address
```

Comment:
Can we be a little clearer about what this check does?

This check denotes whether the third parties used for fraud checks are an authoritative source for data from that country. 

This means if we fail we cannot do an ID Fraud check and provide an `identityFraudScore` for that user.

Reply:
> whether the third parties used for fraud checks are an authoritative source for data from that country.

Got a couple of Qs so I can come to an opinion on the new `permissible_value`. 

If these Qs are answered in business requirements somewhere, apologies, if you could link me I'll go away and read up. 

1. What piece of data are we submitting for a check. 
2. What kind of endpoint tells us the answer - is it polling authoritative sources themselves, or some config or reference that we own?
3. What is the logic of the check? 

Is it something like "user is submitting UK-issued documents, so we're checking that (e.g.) experian is a valid option for KBV against our reference source config?" 

4. What would a VC look like where `fraudCheck: country_data` appears (roughly, don't need full example) - is/are there other attributes that denote a success or fail? 
5. Presumably we wouldn't have both `fraudCheck: country_data` and `fraudCheck: identity_theft_check` in the same VC, so I'm just trying to get clued up on the process. The VC with `fraudCheck: country_data` is done prior to, or same time as, other fraud checks? 

I'm thinking that `country_data` is not as self-descriptive as some of the other `permissible_value` items if somebody doesn't have the above context, but I'm struggling to suggest an alternative without knowing the context. It might be as simple as `country_data_check`.

Reply:
@MikeCollingwood has given me the [ticket](https://govukverify.atlassian.net/browse/PYIC-7650) so I'll read up first then chase with any residual questions

Reply:
I've tried to move the conversation out of the PR and answer some of the questions to the above. We can keep the conversation going on the ticket if that's our preferred non-github focal point?